convert_to_multi_instance.sh

```diff
@@ -165,6 +165,8 @@ function migrate_lic_cms() {
                 yq -i '.metadata.namespace = "'${controlNs}'"' tmp.yaml
                 ${OC} apply -f tmp.yaml
```

Comment:
Could we check the return value of applying new ConfigMap?

I am worried that if it failed to apply new ConfigMap, and directly delete the old one, which leads to a data loss.